src/Microsoft.OData.Core/Microsoft.OData.Core.cs

```diff
@@ -18,7 +18,7 @@ namespace Microsoft.OData
     /// <summary>
     ///    AutoGenerated resource class. Usage:
     ///
-    ///        string s = TextRes.GetString(TextRes.MyIdentifier);
+    ///        string s = TextRes.GetString(TextRes.MyIdenfitier);

```

Comment:
Why does this look like a typo?
`MyIdentifier` looks like the logical spelling

Reply:
+1

---
In reply to: [441495806](https://github.com/OData/odata.net/pull/1793#discussion_r441495806) [](ancestors = 441495806)